src/parameters/AerosolModalNucleation.jl

```diff
@@ -1,14 +1,16 @@
+export H2S04NucleationParameters,
+    OrganicNucleationParameters, MixedNucleationParameters
+
 """
     H2S04NucleationParameters{FT}
 
-Parameters for pure sulfuric acid nucleation
+Parameters for pure sulfuric acid nucleation from Dunne et al 1016
+DOI:10.1126/science.aaf2649
 
 # Fields
 $(DocStringExtensions.FIELDS)
 """
-
-Base.@kwdef struct H2S04NucleationParameters{FT} <:
-                   AbstractCloudMicrophysicsParameters
+Base.@kwdef struct H2S04NucleationParameters{FT} <: ParametersType{FT}
```

Comment:
We are using `@kwdef` for structs at many locations. In most cases, default values are not assigned. A normal `struct` would suffice in these cases!